config.yml

```diff
@@ -18,13 +18,21 @@ blackfriday:
   fractions:            false
 
 publishDir:             "_gh_pages"
-archetypeDir:           "site/archetypes"
-assetDir:               "site/assets"
-contentDir:             "site/content"
-dataDir:                "site/data"
-layoutDir:              "site/layouts"
-staticDir:              "site/static"
-themesDir:              "site/themes"
+
+module:
+  hugoVersion:
+    min:                0.56.3
```

Comment:
Maybe we don't need to set `hugoVersion` at all?

Reply:
That value is mostly for theme creators -- to warn people using your module.

Reply:
Oh, I see, so it doesn't really make sense for us here. I will remove it.